identity-wallet/src/state/connections/mod.rs

```diff
@@ -21,3 +21,12 @@ pub struct Connection {
 
 #[typetag::serde(name = "connection")]
 impl FeatTrait for Connection {}
+
+impl Connection {
+    // TODO: This is a HORRIBLE solution to determine the connection_id by the non-unique "issuer name".
+    // It is a TEMPORARY solution and should only be used in DEMO environments,
+    // since we currently lack a unique identitfier to distinguish connections.
```

Comment:
I think this is a nice opportunity to replace this somewhat screamy TODO message to :
```
// TODO(ngdil): Temporary solution to support NGDIL demo, replace with different unique identifier to distinguish connection
```